app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -93,7 +95,12 @@ private void loadMediaInfo() {
     }
 
     private void displayMediaInfo(@NonNull Playable media) {
-        txtvPodcastTitle.setText(media.getFeedTitle());
+        if (media.getClass() == FeedMedia.class) {
+            String pubDateStr = DateUtils.formatAbbrev(getActivity(), ((FeedMedia) media).getPubDate());
+            txtvPodcastTitle.setText(media.getFeedTitle() + ", " + pubDateStr);
```

Comment:
Just a thought: Would a middle dot like on the queue screen work here, too? Not sure what I like more. (cc @keunes)

Reply:
just pushed a dot as separator

Reply:
Wasn't sure about the middle dot, but in the screenshot above I like it :)
Just wondering how it looks if there's a long podcast title, pushing (half) the date to the next line. Minor issue, though.

Reply:
Maybe replacing the spaces with non-breaking-spaces would be good for that. I think the ones within the date and the ones before/after the dot would be good to replace, so that the dot is never the first or last character in a line

Reply:
Check my latest build around the non-breaking space.  Here is a forced test

![Screen Shot 2021-01-20 at 6 01 41 PM](https://user-images.githubusercontent.com/149837/105269847-9413d500-5b49-11eb-8c21-e9b16710dd1f.png)

Reply:
How about replacing the spaces within the date, too? Otherwise, you could end up with a single number being put in the next line.

Reply:
done